### PR TITLE
Fix some language in contributing and governance

### PR DESCRIPTION
**Pull request by andschwa**
Just caught this while looking through the docs for #2168.

### CODE REVIEW

**Comment by andschwa** [comment]
Thanks @dthaler!

File: docs/Contributing.md [hunk 1]
```diff
@@ -82,15 +82,17 @@ Please do:
 - **DO** submit all code changes via pull requests (PRs) rather than through a
   direct commit. PRs will be reviewed and potentially merged by the repo
   Committers after a peer review that includes at least one Committer.
-- **DO** give PRs short-but-descriptive names (e.g. "Improve code coverage for
-  System.Console by 10%", not "Fix #1234").
+- **DO** give PRs short, but descriptive names (e.g. "Improve code coverage for
```
Comment (CodeMonkeyLeet) [inline docs/Contributing.md]: I don't think you need the comma between the adjectives, it's only used before "but" when introducing an independent clause, which "but descriptive names" is not.

File: docs/Contributing.md [hunk 2]
```diff
@@ -82,15 +82,17 @@ Please do:
...
+  edger8r", not "Fix #1234").
 - **DO** add breaking changes, new features, deprecations, and bug
-  fixes to the [unreleased section of the
-  changelog](../CHANGELOG.md#unreleased).
+  fixes to the [unreleased section of the changelog](../CHANGELOG.md#unreleased).
 - **DO** refer to any relevant issues and include [keywords](
   https://help.github.com/articles/closing-issues-via-commit-messages/) that
   automatically close issues when the PR is merged.
-- **DO** tag any users that should know about and/or review the change.
+- **DO** tag any users that should know about and/or review the change. While
```
Comment (CodeMonkeyLeet) [inline docs/Contributing.md]: Tangent: The use of CODEOWNERS.md actually pulls in a lot of reviewers today for some files and adding even more reviewers can cause them to be lost in the noise. We might want to consider guidance for targeting reviewers using the Assignee field as well.

Comment (andschwa) [reply docs/Contributing.md]: Yeah we need to watch this and identify which sets of files should be setup to who. The initial implementation was kind of just a best guess 😂

**Pull request merged**